recipes/fastq-pair/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "fastq-pair" %}
+{% set version = "1.0" %}
+{% set sha256 = "74fd5bae4d85cc02245ff1b03f31fa3788c50966d829b107076a806ae061da3b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/linsalrob/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - cmake >=3.6
+    - {{ compiler('c') }}
+
+test:
+  files:
```

Comment:
We usually do not include test data in the repo. The command test should be enough.

If you like you can include a `make test` is it exists.

Reply:
Okay, thanks for the advice. I removed the test data and test script. Unfortunately, there's no `make test` for this tool.